src/userCLI/src/main/kotlin/org/imperial/mrc/hint/userCLI/App.kt

```diff
@@ -36,6 +36,19 @@ fun main(args: Array<String>) {
 
     val dataSource = DbConfig().dataSource(ConfiguredAppProperties())
 
+    fun errorOutputStream(msg: String)
+    {
+        try
+        {
+            val resources = ResourceBundle.getBundle("ErrorMessageBundle", Locale("en"))
+            val message = resources.getString(msg.takeLastWhile { it.isLetter() })
```

Comment:
So is this the key line that is fixing the bug? Is the error just being masked here, and if so is it ok to do that?

Reply:
Yes that's the issue, there many ways to fix the issues such as not inheriting HintException but this will cause other issues, see below comment.